frontend/packages/dev-console/src/components/import/serverless-function/AddServerlessFunctionForm.tsx

```diff
@@ -147,10 +147,22 @@ const AddServerlessFunctionForm: React.FC<
                   title={t('devconsole~func.yaml is not present and builder strategy is not s2i')}
                 >
                   <p>
-                    {t(
-                      'devconsole~func.yaml must be present and builder strategy should be s2i to create a Serverless function',
-                    )}
+                    <Trans t={t} ns="devconsole">
+                      The Function cannot be built from this repository since it's not created with{' '}
+                      <code className="co-code">kn func create</code> command.
+                    </Trans>
                   </p>
+                  <a
+                    href={
+                      'https://docs.openshift.com/serverless/1.31/functions/serverless-functions-getting-started.html'
+                    }
+                    target="_blank"
+                    className="co-external-link"
+                    style={{ marginTop: '15px' }}
+                    rel="noopener noreferrer"
```

Comment:
Can you use `ExternalLink` component and add styling in css file and add the classname and use patternfly global variable for spacing ?